src/utils/Types.tsx

```diff
@@ -14,15 +12,15 @@ export type Dimensions = {
   scale: number;
 };
 export type StateProp =
-  | row[]
-  | combinedRow[]
+  | Row[]
+  | CombinedRow[]
   | string
-  | combinedRow
-  | position
+  | Row
+  | Position
   | number
   | boolean;
 
-export interface row {
+export type Row = {
```

Comment:
Changed a lot of interfaces here to types, since I'll use interfaces to pass as props into functional components